float32 Bat1_Volt
float32 Bat1_Curr
float32 Bat1_Power
float32 Bat1_Temp
bool Bat1_Charge
bool Bat1_DisCharge

float32 Bat2_Volt
float32 Bat2_Curr
float32 Bat2_Power
float32 Bat2_Temp
bool Bat2_Charge
bool Bat2_DisCharge

float32 Bat3_Volt
float32 Bat3_Curr
float32 Bat3_Power
float32 Bat3_Temp
bool Bat3_Charge
bool Bat3_DisCharge

float32 Bat4_Volt
float32 Bat4_Curr
float32 Bat4_Power
float32 Bat4_Temp
bool Bat4_Charge
bool Bat4_DisCharge
